src/components/ReviewBox.js

```diff
@@ -2,12 +2,12 @@ import { Link } from 'react-router-dom';
 
 const ReviewBox = (props) => {
     return (
-        <li key={props.title} class="review-box">
+        <li key={props.title} className="review-box">
```

Comment:
You could make this dry-er by de-structuring the elements from props above your return statement.